recipes/tinycbor/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You don't need to include a copy of the license? Isn't it included in the source archive?

Reply:
Correct the LICENCE is provided in tarball.